### PR TITLE
precompute_hiddens/Parser: look up CPU ops once (v4)

### PR DESCRIPTION
**Pull request by danieldk**
## Description

Avoid looking up CPU ops multiple times to get the CBlas struct.

### Types of change

Performance enhancement.

## Checklist

- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

**Comment by svlandeg** [comment]
Should this target the `v4` branch?

**Comment by danieldk** [comment]
> Should this target the `v4` branch?

Yes, I still need to rebase it on that branch.

**Comment by danieldk** [comment]
Cherry-picked on top of `v4` now.

File: spacy/ml/parser_model.pyx [hunk 1]
```diff
@@ -347,6 +347,7 @@ cdef class precompute_hiddens:
     cdef bint _is_synchronized
     cdef public object ops
     cdef public object numpy_ops
+    cdef public object cpu_ops
```
Comment (svlandeg) [inline spacy/ml/parser_model.pyx:350]: Shall we underscore this one like for the `transition_parser`?

**Review by svlandeg — COMMENTED** [review]
Makes sense. Just had a small question on naming consistently, otherwise good to merge!

File: spacy/ml/parser_model.pyx (hunk 1 above)
Comment (danieldk) [reply spacy/ml/parser_model.pyx]: Fixed.

**Comment by danieldk** [comment]
Merged in `v4` to make CI happy.

**Review by svlandeg — APPROVED** [review]

**Pull request merged**